src/components/Campaign.tsx

```diff
@@ -53,19 +57,53 @@ export const Campaign = () => {
           <Header>
             <EditorTopBar
               data-testid={editorTopBarTestId}
-              onSave={save}
               title={
                 campaignContentQuery.data?.campaignName
                   ? campaignContentQuery.data.campaignName
                   : ""
               }
-            />
+            >
+              <ul className="ed-header-list">
+                <li>
+                  {campaignContentQuery.data?.type === "unlayer" ? (
+                    <button
+                      type="button"
+                      onClick={() => setOpen(true)}
+                      className="dp-button button-medium primary-green"
+                    >
+                      <FormattedMessage id="save_template" />
+                    </button>
+                  ) : (
+                    ""
+                  )}
+                </li>
+                <li>
+                  <button
+                    type="button"
+                    onClick={save}
+                    className="dp-button button-medium primary-green"
+                  >
+                    <FormattedMessage id="save" />
+                  </button>
+                </li>
+              </ul>
+            </EditorTopBar>
           </Header>
           <Footer>
             <EditorBottomBar {...continuationUrls}></EditorBottomBar>
           </Footer>
         </>
       )}
+      {campaignContentQuery.data?.type === "unlayer" ? (
+        <SaveAsTemplateModal
+          isOpen={isOpen}
+          content={campaignContentQuery.data}
+          defaultName={campaignContentQuery.data.campaignName}
+          onClose={() => setOpen(false)}
```

Comment:
What do you think about simply naming it `close`?

```suggestion
          close={() => setOpen(false)}
```

Reply:
Is the same for me, I will do change for you.